data/projects/mpz.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  ProjectCategory,
+  ProjectContent,
+  ProjectInterface,
+  ProjectStatus,
+} from "@/lib/types"
+
+const content: ProjectContent = {
+  en: {
+    tldr: "A safe, performant, modular and portable multi-party computation (MPC) library.",
+    description: `mpz is a collection of Rust-based libraries for multi-party computation (MPC), designed to be safe, performant, modular, and easy to use. As the name suggests —mpz is pronounced “em-peasy”— the project aims to make MPC as accessible as possible.
+
+Built with portability in mind, mpz runs natively or in the browser via WebAssembly (WASM). It serves as the core MPC implementation behind [TLSNotary](https://pse.dev/en/projects/tlsn), but its architecture is broadly applicable across privacy-preserving and cryptographic use cases.
```

Comment:
@kalidiagne what's the best practice for internal linking?

Should this be a link to `[TLSNotary](/projects/tlsn)` vs entire URL`[TLSNotary](https://pse.dev/en/projects/tlsn)`? My guess is "yes" 😄